Editor/EditorCore/UVEditor.cs

```diff
@@ -714,7 +714,12 @@ internal bool ClickShortcutCheck(ProBuilderMesh pb, Face selectedFace)
             {
                 return CopyFaceUVSettings(pb, selectedFace);
             }
+#if UNITY_EDITOR_OSX
+            // PBLD-189: Auto-stitch has incorrect and inconvenient shortcut on MacOS
+            else if (em == (EventModifiers.Command | EventModifiers.Shift | EventModifiers.Alt))
```

Comment:
Does not compile currently:

```suggestion
            else if (e.modifiers == (EventModifiers.Command | EventModifiers.Shift | EventModifiers.Alt))
```

Reply:
`Cmd + Shift + Alt` seems to not come through into `IsCopyUVSettingsModifiers` unfortunately on 2022.3 - seems that the view tool activates instead. On the otherhand, `Ctl + Cmd + Shift` does work (ctrl is valid key on macOS btw and doesn't trigger the menu if shift is involved as well).